app/book.py

```diff
@@ -0,0 +1,22 @@
+class Book:
+    def __init__(self, title: str, content: str) -> None:
+        self.title = title
+        self.content = content
+
+    def display(self, display_type: str) -> None:
+        if display_type == "console":
+            print(self.content)
+        elif display_type == "reverse":
+            print(self.content[::-1])
+        else:
+            raise ValueError(f"Unknown display type: {display_type}")
```

Comment:
Not as SOLID
1. display_obj = get_display_by_type(display_type)
2. display_obj.display()

Fix in all places